docs/getting-started/data-views-in-sec.asciidoc

```diff
@@ -0,0 +1,37 @@
+[[data-views-in-sec]]
+= {data-sources-cap} in Elastic Security
+
+The active {data-source} determines what data appears throughout {es-sec} on pages with event or alert data.
+{Data-sources} are defined by the index patterns they include.
+Only data from the {es} {ref}/documents-indices.html[indices], {ref}/data-streams.html[data streams], or {ref}/alias.html[index aliases] specified in the active {data-source} will appear.
+
+IMPORTANT: Custom indices may not be included in the <<default-data-view-security, default {data source}>>. Modify it or create a custom {data-source} to include custom indices.
+
+[discrete]
+== Switch to another {data-source}
+
+You can tell which {data-source} is active by clicking the {data-source-cap} button that appears at the upper right of {es-sec} pages that display data, such as Overview, Alerts, or Hosts.
```

Comment:
> button that appears at the upper right of {es-sec} pages

Not sure if this is the spot to mention it, but Data View's are present within `Timeline`'s as well:

<p align="center">
  <img width="500" src="https://user-images.githubusercontent.com/2946766/171726737-baabf100-78c5-4888-ad9f-ee61cac4e166.png" />
</p>